config/jobs/kubernetes/sig-node/containerd.yaml

```diff
@@ -469,10 +468,7 @@ periodics:
       - --extract=ci/latest
       # Note: The GCE Node image used may have a dependency on the nvidia-driver-installer image defined in https://github.com/kubernetes/kubernetes/blob/master/test/e2e/testing-manifests/scheduling/nvidia-driver-installer.yaml
       # If updating the image defined here, the cos-gpu-installer image may need to updated to support the corresponding COS image.
-      - --env=KUBE_NODE_OS_DISTRIBUTION=gci
-      - --env=KUBE_GCE_MASTER_IMAGE=cos-109-17800-66-33
       - --env=KUBE_GCE_NODE_IMAGE=cos-109-17800-0-47
```

Comment:
We can drop this env too right?

Reply:
I'd not touch nvidia jobs in this PR due to the dependency on nvidia-driver-installer mentioned in the comment on the lines 470-471. I explained this in the description of this PR.

Reply:
Got it. :+1: . Thought the nvidia jobs were only in the gcp/gpu directory.